scripts/bash/run_enceladus.sh

```diff
@@ -285,12 +285,26 @@ case $key in
     shift
     ;;
   *)    # unknown option
-    POSITIONAL+=("$1") # save it in an array for later
+    OTHER_PARAMETERS+=("$1") # save it in an array for later
     shift # past argument
     ;;
 esac
 done
-set -- "${POSITIONAL[@]}" # restore positional parameters
+
+too_many_options_print() {
+  echo "$1: Found unrecognized options passed to the script. Parameters are:"
+  echo "    $2"
+  echo ""
+}
+
+if [[ -n "${OTHER_PARAMETERS[*]}" ]]; then
```

Comment:
I am pretty convinced that this should be executed only if  `[ "$HELP_CALL" == "0" ];` So should be placed on current line 330.

Reply:
When we try and run Git `--help` for example. If there is something specified, it tries to help you with the stuff specified. It even opens man pages for you if you run `git branch --help`. I am not sure we want to offer any services like those. And when you run `git alfa --help` it throws an error that `alfa` is unknown. So, I think this is fine.

And to add to the argument. Our Help executes spark context and runs the spark-jobs. I do not wanna run it more times than needed.

Reply:
I don't know imagine somebody runs a comad with `--version` instead of `--dataset-version`, and adding `--help`...
OK, probably a convoluted counter-examples. Let's keep it like you created it 😉